src/main/resources/avro/io/confluent/examples/streams/pageviewregion.avsc

```diff
@@ -1,6 +1,6 @@
 {"namespace": "io.confluent.examples.streams.avro",
  "type": "record",
- "name": "PageViewStats",
+ "name": "PageViewByRegion",
```

Comment:
Just realized this while double-checking the new PR:  I think the "by region" part in "PageViewByRegion" is not correct because these records are simply (user, page, region) tuples -- they are not keyed by region.

I'd suggest to rename this to "PageViewWithRegion", as this schema is used as the output of a stream-table join that enriches page view events -- (user, page, industry, flags) tuples -- with region information.

Reply:
Renamed it to `PageViewWithRegion`